packages/neos-ui-editors/src/SelectBox/index.spec.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import {mount} from 'enzyme';
+import Enzyme, {mount} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
 import {Provider} from 'react-redux';
 import {createStore} from 'redux';
 import Immutable from 'immutable';
 
 import SelectBoxEditor from './index.js';
 import {WrapWithMockGlobalRegistry, MockDataSourceDataLoader} from '../_lib/testUtils';
 
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
Hey @mstruebing I haven't tried it out, but isn't it possible to move this repetitive piece out into a [Jest setup file](https://facebook.github.io/jest/docs/en/configuration.html#setupfiles-array) to configure the Enzyme adapter only once?

Reply:
Good point, I will check that later because tests are failing currently anyway